server/src/main/java/com/cloud/user/AccountManagerImpl.java

```diff
@@ -358,10 +391,46 @@ public UserVO getSystemUser() {
 
     @Override
     public boolean start() {
+        if (apiNameList == null) {
```

Comment:
Could somehow the list be empty but not null at this point?

Reply:
no, the private apinamelist is not initialised till in this block and nowhere else.